src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
+The native token for the Kensetsu platform is the KEN token, which is used for farming and protocol rewards.
+
+### Key Features - KUSD borrowing (available collateral, parameters). Fee Calculation Process
+
+A Vault is the primary entity which operates on the Kensetsu platform. The Vault has the following parameters;
+
+- **Collateral token** — A token that operates as collateral in borrowing operations. The Kensetsu platform operates with six key tokens: XOR, VAL, TBCD, PSWAP, ETH, and DAI.
+- **Debt in borrowed token** - This shows the amount of debt that the user borrowed. The Kensetsu platform allows borrowing of KUSD tokens pegged to USD.
+- **Available to borrow** - Indicates how much a user can borrow until their vault becomes liquidated.
+- **LTV (loan-to-value)** - The debt ratio to the collateral value in your vault, calculated as a percentage.
+- **Stability Fee** - An annual fee on the amount drawn from your vault, calculated as a percentage.
+- **Liquidation penalty** - a fee applied if the value of your collateral falls below a certain threshold, prompting the sale of assets to cover debt. This penalty increases debt.
+
+### Key actions available for Users:
+
+- **Create a vault** — This operation allows users to create a vault in the Kensetsu platform. Users must deposit some collateral tokens to create a vault. Users can not create empty vaults with collateral tokens, but they can create a vault without borrowing tokens.
+
+The Collateral tokens available in Kensetsu are:
+
+- XOR - `0x0200000000000000000000000000000000000000000000000000000000000000`
+- VAL - `0x0200040000000000000000000000000000000000000000000000000000000000`
+- PSWAP - `0x0200050000000000000000000000000000000000000000000000000000000000`
+- TBCD - `0x02000a0000000000000000000000000000000000000000000000000000000000`
+- ETH - `0x0200070000000000000000000000000000000000000000000000000000000000`
+- DAI - `0x0200060000000000000000000000000000000000000000000000000000000000`
+
+* **Borrowing** - Users borrow KUSD tokens, which are pegged to value of USD. The amount of tokens available to borrow is calculated based on the risk parameters of the Kensetsu platform.
+  Users can borrow tokens in existing vaults if the tokens available to borrow are more than zero.
+
+* **RepayDebt** - Users can repay their borrowed token debt, making the vault healthier and preventing its liquidation. To repay their debt, users must have enough of the borrowed token to repay all fees.
```

Comment:
```suggestion
* **RepayDebt** - Users can repay their borrowed token debt, making the vault healthier and preventing its liquidation. To repay their debt, users must have enough of the borrowed token to repay all fees.

* **Close Vault** - Repays outstanding debt and gets back all deposited collateral. It will fail if the user has not enough KUSD to cover debt. This action destroys CDP.
```